### ISSUE TITLE
does not support VS 2013

### ISSUE THREAD

**Issue by JoeBrockhaus** [shanselman/RestoreAfterReloadVSIX#3]
:'(

**Comment by shanselman** [comment]
I thought this bug was fixed in 2013? I don't think this addin is required anymore.

On Nov 20, 2013, at 10:38 AM, JoeBrockhaus notifications@github.com wrote:

:'(

**Comment by JoeBrockhaus** [comment]
You're right - nevermind. I didn't even try - just part of my install habit to go grab this :-x
Thanks
